CHANGES.md

```diff
@@ -7,6 +7,10 @@ Release 4.2
 
 Features
 --------
+* Significant enhancements to COM support under com.sun.jna.platform.win32.COM.util, - [@dhakehurst](https://github.com/dhakehurst)
+ * Use of interfaces and annotations to provide easier implementation of COM interfaces (uses InvocationHandler). - [@dhakehurst](https://github.com/dhakehurst)
+ * Support for COM event callbacks (this was particularly tricky, very happy I got it to work). - [@dhakehurst](https://github.com/dhakehurst)

```

Comment:
You have an extra period at the end of the text and missing an extra period at the end.

I am sure this looks like small stuff, but we're trying to maintain some consistency across the project with so many contributors. Please don't take it personally.